app/assets/stylesheets/application.scss

```diff
@@ -10,31 +11,82 @@ $font-stack: "PT Mono";
 // $primary-brand-color
 // $secondary-brand-color
 
+h1, h2, h3, h4, h5 {
+  font-family: 'Lora', serif;
+}
+
 $primary-background-color: $pastel-orange; //  pastel-orange
 $secondary-background-color: $light-green; // light-green
 // in overrides
-// $primary-color: #F35F55; //  pastel-red
-// $secondary-color: #335650; //dark-green
+$primary-color: #f35f55; //  pastel-red
+$secondary-color: #335650; //dark-green
 
 // Styles
 body {
-
+  background-color: $smoke;
   font-family: $font-stack;
   font-weight: normal;
 }
 
+a:hover, a:focus {
+  color: white;
+}
+
+.park-index-list-item {
+  background-color: $light-green;
+  list-style-type: none;
+  margin-bottom: 5px;
+  padding: 20px 30px;
+}
+
+.index-details {
+  color: $steel;
+  float: left;
+  font-size: 100%;
+  line-height: 2;
+}
+
+.index-rating p {
+  color: $steel;
+  float: center;
+  font-size: 300%;
+  line-height: 0.6;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.park-info p {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.park-icons img {
+  margin-right: 10px;
+  opacity: 0.6;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.new {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.